spring-boot-project/spring-boot/build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
 	optional("com.atomikos:transactions-jta")
 	optional("com.fasterxml.jackson.core:jackson-databind")
 	optional("com.google.code.gson:gson")
+	optional("com.oracle.ojdbc:ucp")
```

Comment:
Just realized I used the old groupId for Oracle. It would be better replace "com.oracle.ojdbc" with "com.oracle.database.jdbc".

Reply:
A PR is usually created off a branch so that's a bit unfortunate now but if you add an extra commit on your fork of `master` and push, this will update this PR automatically.

Reply:
Ok, done.